### PR TITLE
fix build for linux

### PR DESCRIPTION
**Pull request by AiraYumi**
By combining with pull request #1099, you can build a linux build in a gcc 13 environment.
Probably still needs modification to build with gcc 11 or 12.

### CODE REVIEW

File: indra/cmake/00-Common.cmake [hunk 1]
```diff
@@ -182,6 +182,10 @@ if (LINUX OR DARWIN)
 
   list(APPEND GCC_WARNINGS -Wno-reorder -Wno-non-virtual-dtor )
 
+  if(LINUX)
+    list(APPEND GCC_WARNINGS -Wno-maybe-uninitialized -Wno-misleading-indentation -Wno-stringop-truncation -Wno-unused-value )
```
Comment (Nicky-D) [inline indra/cmake/00-Common.cmake:186]: Most of those are not needed and will be fixed in PR 1099

Comment (AiraYumi) [reply indra/cmake/00-Common.cmake:186]: thank you!

**Comment by Nicky-D** [comment]
I rolled merged this changes into PR 1099

**Comment by AiraYumi** [comment]
Thank you.
This PR is no longer needed and will be closed.

**Pull request closed without merging**